src/core/Akka.NodeTestRunner/Sink.cs

```diff
@@ -31,6 +35,7 @@ public bool OnMessage(IMessageSinkMessage message)
             var resultMessage = message as ITestResultMessage;
             if (resultMessage != null)
             {
+                _logger.Tell(resultMessage.Output);
```

Comment:
In all areas within the `Sink`, we log both to the `Console` and to the Akka.IO Logger - that way in the event that anything goes wrong with the Akka,IO transport we still have a local build log for the node that experienced a failure.